.update/do.py

```diff
@@ -0,0 +1,130 @@
+import click
+import requests
+from pathlib import Path
+
+from cnxcommon import ident_hash
+
+
+here = Path(__file__).parent
+CNX_HOST = 'archive-staging.cnx.org'
```

Comment:
Is this same as or different than [rex-host](https://github.com/openstax/cnx-deploy/pull/1102/files#diff-fdbe56b1ec594bb5ca81b36486001878R106) /`rex_host`? And, it just seems to me like `CNX_HOST` will always point to staging?

Reply:
The rex hostname is different than the cnx hostname. However, this should be pointing at production rather than staging. It's just that the openstax/cnx#348 is not on production yet.